lib/src/component_declaration/built_redux_component.dart

```diff
@@ -96,7 +96,7 @@ abstract class BuiltReduxUiComponent<
 
   @mustCallSuper
   @override
-  void redraw([callback()]) {
+  void redraw([Function callback]) {
```

Comment:
This should be changed back (is this leftover from when it was typed to `Function` previously in react-dart?)